op-e2e/faultproofs/util.go

```diff
@@ -56,6 +59,18 @@ func WithAllocType(allocType config.AllocType) faultDisputeConfigOpts {
 	}
 }
 
+func WithDAC(dacUrl string) faultDisputeConfigOpts {
+	return func(cfg *e2esys.SystemConfig) {
+		if c, ok := cfg.Nodes["sequencer"]; ok {
+			c.DACConfig = &node.DACConfig{URLS: []string{dacUrl}}
+			c.Driver = driver.Config{SequencerEnabled: true}
+			c.Rollup.L2BlobConfig = &rollup.L2BlobConfig{
```

Comment:
It's not good abstraction to put it here, you can follow how `rollupAltDAConfig` works [here](https://github.com/ethstorage/optimism/blob/32380c23aa610429e82cf0e9f323e79e364687eb/op-e2e/system/e2esys/setup.go#L613).